packages/core/platform/platform.js

```diff
@@ -55,6 +55,9 @@ const platforms = [
 	{platform: 'webos', regex: /Web0S;.*Safari\/538.2/, forceVersion: 2},
 	{platform: 'webos', regex: /Web0S;.*Chrome\/38/, forceVersion: 3},
 	{platform: 'webos', regex: /Web0S;.*Chrome\/53/, forceVersion: 4},
+	{platform: 'webos', regex: /Web0S;.*Chrome\/68/, forceVersion: 5},
+	// LG webOS of indeterminate version
+	{platform: 'webos', regex: /Web0S;/, forceVersion: -1},
```

Comment:
Would `'unknown'` be better than `-1`?

Reply:
Depends.  We do want `platform.webos` to be truthy for conditional usage and whatnot. Introducing a string value seems odd given we've stuck to integers up til now (even using `0` for legacy webos),

Reply:
speaking of that, since `0` is falsy, it'll appear that the platform is 'legacy' but it't not totally clear that it's legacy webOS.  I wonder if `-1` works better in that case?

Reply:
I have no problem with that. Thoughts @ryanjduffy ?

Reply:
I think the "truthy" check is important so should probably use -1 for any unknown or legacy version. We could add an "extras" field for the fallback if we felt it useful as we have for legacy.